attribute/imageattribute.go

```diff
@@ -0,0 +1,25 @@
+package attribute
+
+import (
+	"github.com/getyoti/yoti-go-sdk/anchor"
+)
+
+//ImageAttribute is a Yoti attribute which returns an image as its value
+type ImageAttribute struct {
```

Comment:
stutter... the fix in this case is to create an image sub pkg... it is common for a go pkg to be light weight.

Reply:
Actually ignore that comment, think there is a better way...